rest/data/patch.go

```diff
@@ -79,6 +79,36 @@ func (pc *DBPatchConnector) FindPatchById(patchId string) (*restModel.APIPatch,
 	return &apiPatch, nil
 }
 
+func (pc *DBPatchConnector) FindPatchesByIds(patchIds []string) ([]restModel.APIPatch, error) {
+	mgobsonIds := []mgobson.ObjectId{}
+	for _, patchID := range patchIds {
+		if err := validatePatchID(patchID); err != nil {
+			return nil, errors.Wrap(err, "problem validating patchId")
+		}
+		mgobsonIds = append(mgobsonIds, mgobson.ObjectIdHex(patchID))
+	}
+
+	p, err := patch.Find(patch.ByIds(mgobsonIds))
+	if err != nil {
+		return nil, err
```

Comment:
I would wrap this error similar to what you do on line 86. The error itself will be a pretty cryptic database error, so wrapping it will add some context to that